sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DocumentAnalysisLiveTestBase.cs

```diff
@@ -116,10 +116,9 @@ protected DocumentModelAdministrationClient CreateDocumentModelAdministrationCli
         /// </summary>
         /// <param name="modelId">The identifier of the model.</param>
         /// <param name="containerType">The type of container to use for training.</param>
-        /// <param name="buildMode">The technique to use to build the model. Defaults to <see cref="DocumentBuildMode.Template"/>.</param>
```

Comment:
Removing this parameter from this internal test helper method since we don't use it. Keeping it simple.